docs/tutorial.rst

```diff
@@ -1213,7 +1217,7 @@ The ``job_scheduler_preamble`` contains the backend job scheduler directives tha
 The ``prepare_cmds`` are commands that can be emitted before the test environment commands.
 These can be specified with the :attr:`~config.systems.partitions.prepare_cmds` partition configuration option.
 The ``env_load_cmds`` are the necessary commands for setting up the environment of the test.
-These include any modules or environment variables set at the `system partition level <config_reference.html#system-partition-configuration>`__ or any `modules <regression_test_api.html#reframe.core.pipeline.RegressionTest.modules>`__ or `environment variables <regression_test_api.html#reframe.core.pipeline.RegressionTest.variables>`__ set at the test level.
+These include any modules or environment variables set at the :ref:`system partition level <system-partition-configuration>` or any :attr:`modules <reframe.core.pipeline.RegressionTest.modules>` or :attr:`environment variables <reframe.core.pipeline.RegressionTest.variables>` set at the test level.
```

Comment:
Use the original formatting for the `:attr:` links (see comment above about the rationale).